src/applications/MachineShopSimulator.java

```diff
@@ -15,7 +15,7 @@
     private int numMachines; // number of machines
     private int numJobs; // number of jobs
     private EventList eList; // pointer to event list
-    private Machine[] machine; // array of machines
+    Machine[] machine; // array of machines

```

Comment:
In general you don't want to be making `private` things non-private unless there's a clear reason to do so. Is there a reason you're doing that here?

Reply:
No I don't think there is. I think that a refactoring tool may have changed this and I didn't notice? Changing it back to private.

Reply:
Update: I had made this non-private as it is being used in the Job class now.

Reply:
Another update after confusion: this is now private with a getter that is used in the method in the Job class.